### DISCUSSION TITLE
Welcome to FOREST discussions!

### DISCUSSION THREAD

**Discussion by andrewgryan** [MetOffice/forest#496] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by claretandy** [comment]
Hi Andy! Good to see some significant re-_forest_ation going on! I've got one eye on using it for visualising climate data (specifically climate impacts indices), but I'm also interested to see how FOREST is growing for the visualisation of regional NWP models and observations. Is there an implementation of FOREST (even if it's just for NWP) that you can share so that we can see all the recent improvements you and Kevin are making?

**Reply by andrewgryan** [reply]
Hi Andy, there's nothing running at the moment but there will be in the coming weeks. I'm keen on simplifying the system from a user perspective so having more users is welcome.

**Reply by andrewgryan** [reply]
I'm also interested in trying different data types with it. Radar seems to be an interesting observation type to add support for.
